aucoalesce/normalizations.yaml

```diff
@@ -45,708 +71,1076 @@ macros:
+      secondary: [addr, hostname]
+      what: user-session
+    record_types: CRYPTO_KEY_USER
+    source_ip: [addr]
+    ecs: *ecs-process
+  - action: crypto-officer-logged-in
+    record_types: CRYPTO_LOGIN
+  - action: crypto-officer-logged-out
+    record_types: CRYPTO_LOGOUT
+    ecs: *ecs-process
+  - <<: *macro-user-session
+    action: started-crypto-session
+    object:
+      primary: addr
+      secondary: [rport]
+    record_types: CRYPTO_SESSION
+    source_ip: [addr]
+    ecs: *ecs-process
+  - action: access-result
+    record_types: DAC_CHECK
+
+  # Anomalies
+
+  # AUDIT_ANOM_ABEND - Process ended abnormally
+  - record_types: ANOM_ABEND
+    action: crashed-program
+    object:
+      primary: [comm, exe]
+      secondary: pid
+      what: process
+    how: sig
+    ecs:
+      # consider adding an anomaly category when we introduce
+      # to ECS
+      <<: *ecs-process
+      type: end
+  # AUDIT_ANOM_EXEC - Execution of file
+  - record_types: ANOM_EXEC
+    action: attempted-execution-of-forbidden-program
+    object:
+      primary: cmd
+      what: process
+    how: terminal
+    ecs:
+      # consider adding an anomaly category when we introduce
+      # to ECS
+      <<: *ecs-process
+      type: start
+  # AUDIT_ANOM_LINK - Suspicious use of file links
+  - record_types: ANOM_LINK
+    action: used-suspcious-link
+  # AUDIT_ANOM_LOGIN_FAILURES - Failed login limit reached
+  - <<: *macro-user-session
+    record_types: ANOM_LOGIN_FAILURES
+    action: failed-log-in-too-many-times-to
+  # AUDIT_ANOM_LOGIN_LOCATION - Login from forbidden location
+  - <<: *macro-user-session
+    record_types: ANOM_LOGIN_LOCATION
+    action: attempted-log-in-from-unusual-place-to
+  # AUDIT_ANOM_LOGIN_SESSIONS - Max concurrent sessions reached
+  - <<: *macro-user-session
+    record_types: ANOM_LOGIN_SESSIONS
+    action: opened-too-many-sessions-to
+  # AUDIT_ANOM_LOGIN_TIME - Login attempted at bad time
+  - <<: *macro-user-session
+    record_types: ANOM_LOGIN_TIME
+    action: attempted-log-in-during-unusual-hour-to
+  # AUDIT_ANOM_PROMISCUOUS - Device changed promiscuous mode
+  - record_types: ANOM_PROMISCUOUS
+    action: changed-promiscuous-mode-on-device # Could be entered or exited based on prom field.
+    object:
+      primary: dev
+      what: network-device
+  # AUDIT_ANOM_RBAC_INTEGRITY_FAIL - RBAC file integrity failure
+  - record_types: ANOM_RBAC_INTEGRITY_FAIL
+    action: tested-file-system-integrity-of
+    object:
+      primary: hostname
+      what: filesystem
+  # AUDIT_ANOM_LOGIN_ACCT - Login attempted to watched acct
+  # AUDIT_ANOM_MAX_DAC - Max DAC failures reached
+  # AUDIT_ANOM_MAX_MAC - Max MAC failures reached
+  # AUDIT_ANOM_AMTU_FAIL - AMTU failure
+  # AUDIT_ANOM_RBAC_FAIL - RBAC self test failure
+  # AUDIT_ANOM_CRYPTO_FAIL - Crypto system test failure
+  # AUDIT_ANOM_MK_EXE - Make an executable
+  # AUDIT_ANOM_ACCESS_FS - Access of file or dir
+  # AUDIT_ANOM_ADD_ACCT - Adding an acct
```

Comment:
So, we still have a bunch of ECS normalization we can add for some of these audit types, but I figured I could do that in a subsequent PR since this is already a ton of changes